kotlin-mui-material/src/jsMain/generated/mui/material/Portal.kt

```diff
@@ -0,0 +1,20 @@
+// Automatically generated - do not modify!
+
+@file:JsModule("@mui/material/Portal")
+
+package mui.material
+
+/**
+ * Portals provide a first-class way to render children into a DOM node
+ * that exists outside the DOM hierarchy of the parent component.
+ *
+ * Demos:
+ *
+ * - [Portal](https://mui.com/material-ui/react-portal/)
+ *
+ * API:
+ *
+ * - [Portal API](https://mui.com/material-ui/api/portal/)
+ */
+@JsName("default")
```

Comment:
In previous releases it was removed (`default`)?

Reply:
No. It was removed in MUI Base, but not in MUI Material